Code/Tools/LuaIDE/Source/Utils/LuaIDEConstants.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Name/Name.h>
+
+namespace LUADebugger
+{
+    static const AZ::Name LuaToolsName = AZ::Name("LuaRemoteTools");
+    static const AZ::Crc32 LuaToolsKey = AZ_CRC(LuaToolsName.GetCStr());
```

Comment:
AZ_CRC isn't needed here and doesn't provide a benefit.

actually the constructor should be used directly should be used here for compile time generation
```suggestion
    static constexpr AZ::Crc32 LuaToolsKey("LuaRemoteTools");
```